### PR TITLE
Use shootPenalty skill with geneva fixed

### PR DESCRIPTION
**Pull request by RobZelluf**

### CODE REVIEW

File: roboteam_ai/src/control/shotControllers/ShotController.cpp [hunk 1]
```diff
@@ -63,7 +68,7 @@ RobotCommand ShotController::getRobotCommand(world::Robot robot, const Vector2 &
             if (robot.hasWorkingBallSensor()) {
                 shotData = shoot(robot, lineToDriveOver, aimTarget, chip, ballspeed);
             } else {
-                shotData = shootWithoutBallSensor(robot, lineToDriveOver, aimTarget, chip, ballspeed);
+                shotData = shoot(robot, lineToDriveOver, aimTarget, chip, ballspeed);
```
Comment (ThijsLuttikhuis) [inline roboteam_ai/src/control/shotControllers/ShotController.cpp]: should this be shoot as well??

**Review by ThijsLuttikhuis — APPROVED** [review]

File: roboteam_ai/src/control/shotControllers/ShotController.cpp (hunk 1 above)
Comment (mrlukasbos) [reply roboteam_ai/src/control/shotControllers/ShotController.cpp]: This should indeed be changed back

File: roboteam_ai/src/skills/gotopos/GoToPos.cpp [hunk 2]
```diff
@@ -76,7 +76,7 @@ bt::Node::Status GoToPos::onUpdate() {
             return Status::Success;
         }
     }
-    if (command.x_vel == 0 && command.y_vel == 0 && command.w == 0) {
+    if (command.x_vel == 0 || command.y_vel == 0 || command.w == 0) {
```
Comment (mrlukasbos) [inline roboteam_ai/src/skills/gotopos/GoToPos.cpp]: why is this change?

Comment (RobZelluf) [reply roboteam_ai/src/skills/gotopos/GoToPos.cpp]: Because it otherwise breaks

**Pull request merged**